lib/signs_ui_web/controllers/auth_controller.ex

```diff
@@ -0,0 +1,25 @@
+defmodule SignsUiWeb.AuthController do
+  use SignsUiWeb, :controller
+  plug(Ueberauth)
+
+  @spec callback(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def callback(%{assigns: %{ueberauth_auth: auth}} = conn, _params) do
+    username = auth.uid
+    expiration = auth.credentials.expires_at
+
+    current_time = DateTime.utc_now() |> DateTime.to_unix()
```

Comment:
`current_time = System.system_time(:second)`

Reply:
Updated here and elsewhere in the PR where `DateTime.utc_now()` was used.